google/cloud/pubsub/admin/subscription_admin_client.h

```diff
@@ -0,0 +1,925 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Generated by the Codegen C++ plugin.
+// If you make any local changes, they will be lost.
+// source: google/pubsub/v1/pubsub.proto
+
+#ifndef GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_PUBSUB_ADMIN_SUBSCRIPTION_ADMIN_CLIENT_H
+#define GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_PUBSUB_ADMIN_SUBSCRIPTION_ADMIN_CLIENT_H
+
+#include "google/cloud/future.h"
+#include "google/cloud/options.h"
+#include "google/cloud/polling_policy.h"
+#include "google/cloud/pubsub/admin/subscription_admin_connection.h"
+#include "google/cloud/status_or.h"
+#include "google/cloud/version.h"
+#include <map>
+#include <memory>
+
+namespace google {
+namespace cloud {
+namespace pubsub_admin {
+GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
+
+///
+/// The service that an application uses to manipulate subscriptions and to
+/// consume messages from a subscription via the `Pull` method or by
+/// establishing a bi-directional stream using the `StreamingPull` method.
```

Comment:
I see what you mean.  Yes, we need to figure out a way to replace the documentation for this class.

Reply:
https://github.com/googleapis/google-cloud-cpp/pull/12982